src/Stratis.Bitcoin.Features.Api/ApiFeature.cs

```diff
@@ -107,6 +112,12 @@ public override void Dispose()
 
     public sealed class ApiFeatureOptions
```

Comment:
This class name doesn't seem to match what its doing.  If it's going to include more API type options then fine, if it's only going to deal with Certification then maybe it needs to be renamed.

Reply:
OK I am not too fussed about what the name is/was, I think this is a certificate that is only used for the ApiFeature so I am not shocked to have its location in the ApiFeatureOptions, do you have a better name ?

Reply:
Ar sorry I thought this was new.  If it exists then not to worry.